### ISSUE TITLE
Jest for jquery testing

### ISSUE THREAD

**Issue by nellysattari** [kentcdodds/ama#435]
I am looking for testing my js files with JEST which contain JQUERY code as well. How I can do it with JEST? 
Thanks a lot

**Comment by ajwhite** [comment]
If the challenge is exposing the library to your test environment, https://github.com/facebook/jest/issues/708#issuecomment-339418577 may be helpful

**Comment by nellysattari** [comment]
Thanks @ajwhite  I see how to import Jquery. But not sure if you have the option to test the value or attributes of Juery element which is set through some Javascript functions.

**Comment by kentcdodds** [comment]
Hi! I'm afraid I don't have time to give a super solid answer to all my open questions right now, so I just went through and answered all my AMA questions in the form of a video: https://www.youtube.com/watch?v=y23g8L8UtoQ&list=PLV5CVI1eNcJgCrPH_e6d57KRUTiDZgs0u

A few things that folks can do to help make things better for others reading the questions and answers:

1. Link to when I start answering the question. Add `&t=xxmxxs` to the end of the URL like this: https://www.youtube.com/watch?v=y23g8L8UtoQ&list=PLV5CVI1eNcJgCrPH_e6d57KRUTiDZgs0u&t=2m5s
2. Comment with links I referenced
3. Summarize what I talk about

Thanks!
